### ISSUE TITLE
Unable to start Dev server. 

### ISSUE THREAD

**Issue by flipsyde606** [ConsenSys/truffle-webpack-demo#11]
## Issue

Unable to start dev server. For some reason it doesn't seem to find truffle.js config

## Steps to Reproduce

$ WEB3_RPC_LOCATION=http://localhost:8545 npm run start

## Expected Behavior

App to start with default settings. 

## Actual Results

I am getting the below log

$ WEB3_RPC_LOCATION=http://localhost:8545 npm run start

> truffle-webpack@0.0.1 start E:\Dev\Digirupt\Code
> node ./scripts/start.js

Starting the development server...

[TRUFFLE SOLIDITY] No Truffle config file found.
[TRUFFLE SOLIDITY] Setting network to 'default' for compilation and contract provisioning
[TRUFFLE SOLIDITY] Setting network_id to 'default' for compilation and contract provisioning
Failed to compile.

Error in ./contracts/MetaCoin.sol
Module build failed: Error: You must specify the location of the Truffle migrations directory in the loader query string. (migrations_directory)
 @ ./src/components/AccountList/AccountListContainer.js 33:16-49

## Environment

* Operating System: Win 10 Home (x64)
* Truffle version:  2.1.1
* Testrpc version: 3.0.1
* Ethereum client: geth
* node version: 6.9.1
* npm version: 3.10.9

**Comment by johnmcdowall** [comment]
This is probably a windows path issue. Unfortunately I dont have access to a Windows machine, help wanted!

**Comment by flipsyde606** [comment]
Yep it probably is. Meanwhile, I was able to get this running this over Ubuntu via vagrant. Had to make changes in start.js to bind IP and port though. I think it would be nice if port and ip are not hard coded as localhost:3000.

**Comment by DiscRiskandBisque** [comment]
I found the solution! In the [truffle-solidity docs](https://github.com/ConsenSys/truffle-solidity-loader) it mentioned the truffle.js file should be named truffle-config.js. I renamed the config file and also changed the name in index.js.

At this point the deployment script ran out of gas, so I increased it from 190000 to 300000 and the contracts deployed successfully.

**Comment by sun337** [comment]
I think i have a similar issue. `npm start` gives the following 

`Starting the development server...

Failed to compile.

Error in ./contracts/MetaCoin.sol
Module build failed: Error: You must specify the network name to deploy to. (network)
 @ ./src/components/AccountList/AccountListContainer.js 37:16-49`

Upgraded from truffle-2 to truffle-3 recently.
I'm using 
```
truffle-solidity-loader: "git+https://github.com/sogoiii/truffle-solidity-loader.git#1f1e213d52f033b6863218307b8968ae68220fe1"
truffle: 3.2.1
webpack-dev-server: 2.4.1
webpack: 2.2.1
```
